src/environments/environment-interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface Environment {
+  production: boolean
```

Comment:
Boolean is quite limitting. 
Already now we have three different environments: `Production`, `Staging` and `Development`

Reply:
`production` was already defined. And do we intend to make builds specific to a staging environment where the configuration is different from production?

Reply:
I don't think so, but using a boolean for this is still vague.
What environment are we in when `production = false`? It could be anything.

Reply:
But I think that an argument could be made that we shouldn't include our environment files in VCS at all. We can't store keys (and production URLS) in the environment when we publish them to Git.

Reply:
I just checked it out and `production` is a variable that Angular itself uses.